api/src/schemas.ts

```diff
@@ -146,6 +146,21 @@ export const schemas = {
       })
     }
   },
+  getSessionUser: {
+    response: {
+      200: Type.Object({
+        // Unfortunately, it's not possible to set a real schema for this
+        // object, since the username is dynamic and used as a key.
+        user: Type.Any(),
```

Comment:
I'll see if I can come up with something better, but the fundamental problem is that we have a silly data-structure that doesn't neatly map onto JSON schemas.

Or... I'm doing it wrong. I'll see if I can do it right.

Reply:
We might be able to hack something together with `Type.String({ maxLength: 1024 })` and `Type.Record`.

Maybe. It's going to be nasty, though.

Reply:
Alrighty. A record does seem to work as we want, but it's tripping the schema security tests.

I see a few ways forward

1. Give up type safety for now
2. Don't test this schema (the regex for the keys is just `^(.*)$` which seems safe to me)
3. Fix the api-server

I'm leaning heavily towards 2. since 1. is annoying and 3. is a biggish job.

Reply:
We did discuss doing 3 many years ago, but I am also just fine with 2. Go with 2.

Although, I am super surprised this is not already an issue/feature-request on typebox.

Reply:
I did a little digging: you can, effectively, set maxLength by using `Type.String({ pattern: regexOfYourChoice })`. So, technically it can be done.

I played around a bit with setting `maxLength` explicitly, too, but that appears to only be valid for strings not patterns.

Anyways, I'll make an exception for this schema.